charts/lightstepsatellite/values.yaml

```diff
@@ -78,13 +107,15 @@ lightstep:
 
   # Single Project Mode details
   disable_access_token_checking: false
+  # -- REQUIRED if `lightstep.disable_access_token_checking` is `true`
   project_name: ""
 
   # Optional configs
 
   collector_pool: my-satellite-pool
-  bytes_per_project: "1000000000"
+  bytes_per_project: "4000000000"
```

Comment:
This was a deliberate decision to leave at 1GB. Since the satellite by default does not run in single project mode, leaving the value low at 1GB ensures that if you report multiple projects to the same pool, then each project is only assigned a maximum 1GB of memory for span storage. Making this value 4GB means that if you report even two proejects (dev and qa) then you are going to potentially run into OOMKills.

Reply:
Would you be able to replace your bytes_per_project line with:
  #bytes_per_project is intentionally left lower than the recommended 4GB to protect for cases where the satellite handles multiple projects in the same pool. If you are reporting only a single project or using single project mode, increase this to 400000000
  bytes_per_project: "1000000000"